contracts/identity/Identity.sol

```diff
@@ -115,6 +146,15 @@ contract Identity is IdentityAdminRole, SchemeGuard {
         emit BlacklistRemoved(account);
     }
 
+    function _addClaimer(address account) internal {
+        claimers.add(account);
+        
+        increaseClaimerCount(1);
```

Comment:
this will increase even if address is added twice or if address is transfered

Reply:
claimers.add will revert if the given account is already added, and will therefore not increase.
In the case of transfer removeClaimer will run first, decrementing the amount of claimers before it increases when the new user is added